### PR TITLE
Enhancement layerinfo style

### PR DESCRIPTION
**Pull request by alsturm**
add responsive style for layerinfo.

### CODE REVIEW

File: test/modules/geoResourceInfo/components/GeoResourceInfoPanel.test.js [hunk 1]
```diff
@@ -14,24 +15,35 @@ describe('GeoResourceInfoPanel', () => {
...
+				notification: null
+			},
+			media: {
+				portrait: false
+			},
+			...state
+		};
+
+		store = TestUtils.setupStoreAndDi(initialState, { notifications: notificationReducer, media: createNoInitialStateMediaReducer() });
+		// TestUtils.setupStoreAndDi();
```
Comment (taulinger) [inline test/modules/geoResourceInfo/components/GeoResourceInfoPanel.test.js:32]: Can we remove this line?

File: src/modules/geoResourceInfo/components/GeoResourceInfoPanel.js [hunk 2]
```diff
@@ -4,7 +4,9 @@ import { MvuElement } from '../../MvuElement';
 import { $injector } from '../../../injection';
 import { GeoResourceInfoResult } from '../services/GeoResourceInfoService';
 import { emitNotification, LevelTypes } from '../../../store/notifications/notifications.action';
+import css from './geoResourceInfoPanel.css';
 
+const Update_IsPortrait = 'update_isPortrait_hasMinWidth';
 const UPDATE_GEORESOURCEINFO = 'UPDATE_GEORESOURCEINFO';
 
 /**
```
Comment (taulinger) [inline src/modules/geoResourceInfo/components/GeoResourceInfoPanel.js:12]: @author sturmulus is missing

**Review by taulinger — COMMENTED** [review]
Very nice enhancement!

File: src/modules/geoResourceInfo/components/GeoResourceInfoPanel.js (hunk 2 above)
Comment (alsturm) [reply src/modules/geoResourceInfo/components/GeoResourceInfoPanel.js:12]: done

File: test/modules/geoResourceInfo/components/GeoResourceInfoPanel.test.js (hunk 1 above)
Comment (alsturm) [reply test/modules/geoResourceInfo/components/GeoResourceInfoPanel.test.js]: done

**Review by taulinger — APPROVED** [review]

**Pull request merged**